middleware/limiter/limiter_test.go

```diff
@@ -15,9 +15,6 @@ import (
 
 // go test -run Test_Limiter_Concurrency_Store -race -v
 func Test_Limiter_Concurrency_Store(t *testing.T) {
-	t.Parallel()
```

Comment:
can you add a comment to the header of the function where you have removed parallel execution to make it more stable
make a comment in the header of the function

so that nobody gets the idea that it was a mistake and adds it again

Reply:
Added t.Parallel() again, issue was the `go singleRequest()` call doing `require` calls in a goroutine.